scopes/scope/update-dependencies/update-dependencies.main.runtime.ts

```diff
@@ -96,6 +96,11 @@ export class UpdateDependenciesMain {
     // an error saying "the extension ${extensionId.toString()} is missing from the flattenedDependencies"
     // if (!updateDepsOptions.simulation) {
     await this.snapping._addFlattenedDependenciesToComponents(this.legacyComponents);
+    await Promise.all(
+      this.legacyComponents.map(async (component) => {
+        component.dependenciesGraph = await this.scope.legacyScope.getDependenciesGraphByComponentId(component.id);
+      })
```

Comment:
use the new deps resolver addDepsGraph API (see comment on isolator file)

Reply:
but this one is not calculating the graph from the lockfile. This reads the graph from the version object.

Reply:
added loadDependenciesGraphForComponent